### DISCUSSION TITLE
Prompting with null entries

### DISCUSSION THREAD

**Discussion by brknsoul** [adieyal/sd-dynamic-prompts#408] [Q&A] [answered]
Say I want a subject with either messy hair, long hair, short hair, or 'default' hair, would this work?  

`{messy hair|short hair|long hair|}`

Notice the empty string after 'long hair'.
Essentially, would this give me 75% chance of modified hair, and 25% chance of whatever hair the model chooses?

**Comment by akx** [comment, marked as answer]
Yes, exactly.

**Reply by brknsoul** [reply]
Excellent. Thank you!
